node/silkworm/common/stopwatch.hpp

```diff
@@ -28,59 +28,42 @@
 #include <vector>
 
 namespace silkworm {
-/// <summary>
-/// This class mimics the behavior of a stopwatch to measure timings of operations
-/// </summary>
+//! \brief This class mimics the behavior of a stopwatch to measure timings of operations
 class StopWatch {
   public:
     typedef std::chrono::time_point<std::chrono::high_resolution_clock> TimePoint;
     typedef std::chrono::nanoseconds Duration;
```

Comment:
Not related to this change... just wanted to suggest using `using` instead of `typedef`. The benefit is that the new type is right at the beginning and you don't have to hunt for it.

Reply:
Thanks. Done